ai/states/game_state.py

```diff
@@ -19,7 +19,9 @@ def __init__(self):
         initialise le GameState, initialise les variables avec des valeurs nulles
         """
         self.logger = logging.getLogger(self.__class__.__name__)
+        self.reset()
 
+    def reset(self):
```

Comment:
should we do a full reset? some stuff are not reset like teams

Reply:
Good catch